RadarSDK/RadarLocationManager.m

```diff
@@ -396,6 +396,19 @@ - (void)updateTrackingFromMeta:(RadarMeta *_Nullable)meta {
     }
 }
 
+- (void)restartPreviousTrackingOptions {
+    RadarTrackingOptions *previousTrackingOptions = [RadarSettings previousTrackingOptions];
+    [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug message:@"Restarting previous tracking options"];
+
+    if (previousTrackingOptions) {
+        [Radar startTrackingWithOptions:previousTrackingOptions];
+    } else {
+        [Radar stopTracking];
```

Comment:
@corypisano @nickpatrick 

So one thing I notice about the `restartPreviousTrackingOptions` function is that it's called from a few places (cancel, complete, and track) and it's not resilient to multiple of those being called.  The second call will call `stopTracking`.  Maybe this is fine in the happy path but I could imagine it causing issues.  Maybe this is something we're fine with for this release and we iterate on it in a future version.